### PR TITLE
Updates README with minor changes

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**
## Category

- [ ] Bugfix
- [ ] Enhancement
- [ ] New Feature
- [ ] Configuration Change
- [ ] Versioning Change
- [ ] Unit Test(s)
- [ ] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Snapshots are deployed off of the 'master' branch to [OSSRH](https://oss.sonatyp
 https://oss.sonatype.org/content/repositories/snapshots/
 ```
 
-You'll also need:
```
Comment (vijetmahabaleshwar-okta) [inline README.md:95]: To maintain consistency, as we have used `You will` in other sentences.

File: README.md [hunk 2]
```diff
@@ -104,8 +104,8 @@ Construct a client instance by passing it your Okta domain name and API token:
 [//]: # (method: createClient)
 ```java
 Client client = Clients.builder()
-    .setOrgUrl("{yourOktaDomain}")
-    .setClientCredentials(new TokenClientCredentials("{apiToken}"))
```
Comment (vijetmahabaleshwar-okta) [inline README.md:108]: I feel having these brackets is confusing for some users, who tend to include them while adding values. I've seen users ask questions about it on devforum.

File: README.md [hunk 3]
```diff
@@ -411,11 +411,9 @@ okta:
       username: null
       password: null
     authorizationMode: "PrivateKey"
-    clientId: "{clientId}"
-    scopes:
-      - okta.users.read
-      - okta.apps.read
-    privateKey: "~/.okta/privateKey.pem" # PEM format. This SDK supports RSA AND EC algorithms - RS256, RS384, RS512, ES256, ES384, ES512.
+    clientId: "yourClientId"
+    scopes: "okta.users.read okta.apps.read"
```
Comment (vijetmahabaleshwar-okta) [inline README.md:415]: Right now, multi line scopes is not working as expected. @arvindkrishnakumar-okta is aware of the issue. Once he fixes it, we can update the README if needed.

File: README.md [hunk 4]
```diff
@@ -118,16 +118,16 @@ Okta allows you to interact with Okta APIs using scoped OAuth 2.0 access tokens.
...
 This SDK supports this feature only for service-to-service applications. Check out [our guides](https://developer.okta.com/docs/guides/implement-oauth-for-okta/overview/) to learn more about how to register a new service application using a private and public key pair.
 
-When using this approach you won't need an API Token because the SDK will request an access token for you. In order to use OAuth 2.0, construct a client instance by passing the following parameters:
+When using this approach, you won't need an API Token because the SDK will request an access token for you. In order to use OAuth 2.0, construct a client instance by passing the following parameters:
 
 [//]: # (method: createOAuth2Client)
 ```java
 Client client = Clients.builder()
-    .setOrgUrl("{yourOktaDomain}")
+    .setOrgUrl("https://{yourOktaDomain}")  // E.g https://dev-123456.okta.com
```
Comment (arvindkrishnakumar-okta) [inline README.md:126]:
```suggestion
    .setOrgUrl("https://{yourOktaDomain}")  // e.g. https://dev-123456.okta.com
```

File: README.md [hunk 5]
```diff
@@ -104,8 +104,8 @@ Construct a client instance by passing it your Okta domain name and API token:
 [//]: # (method: createClient)
 ```java
 Client client = Clients.builder()
-    .setOrgUrl("{yourOktaDomain}")
-    .setClientCredentials(new TokenClientCredentials("{apiToken}"))
+    .setOrgUrl("https://{yourOktaDomain}")  // E.g https://dev-123456.okta.com
```
Comment (arvindkrishnakumar-okta) [inline README.md:107]:
```suggestion
    .setOrgUrl("https://{yourOktaDomain}")  // e.g. https://dev-123456.okta.com
```

File: README.md (hunk 3 above)
Comment (arvindkrishnakumar-okta) [reply README.md:415]: @vijetmahabaleshwar-okta The issue you noted here was already noticed and fixed (see https://github.com/okta/okta-sdk-java/pull/379). Please build the latest SDK from master and inject it into your client app. You will not see this error.

**Pull request merged**